debug_server/src/server.rs

```diff
@@ -423,6 +434,8 @@ impl Server {
 					vars.push(self.value_to_variable(String::from(name), &local));
 				}
 
+				vars.sort_by_key(|a| a.name.to_lowercase());
```

Comment:
Hmm, I'm not sure if locals should be sorted. The natural order is the declaration order which is at least somewhat useful, compared to the not-useful default order of datum vars.

Reply:
I didn't know that, seems like a decent reason to keep them unsorted, especially as we don't see hundreds of variables in locals. Any preference @willox?

Reply:
I only didn't mention it because I thought space man liked them sorted. I think it's best to only sort fields.